crates/matrix-sdk-crypto-ffi/src/olm.udl

```diff
@@ -15,7 +15,7 @@ interface MigrationError {
 };
 
 callback interface Logger {
-    void log(string log_line);
+    void log(string logLine);
```

Comment:
For some reason, without this change this specific line gets compiled to swift as `log_line` (i.e. does not convert to camelcase), which the Swift compiler does not like and fails. Easy solution for now is to use camelcase directly in the udl.